Parser/pegen.c

```diff
@@ -2735,6 +2770,7 @@ deal_with_gstring2(Parser *p, Token* a, asdl_expr_seq* raw_expressions, Token*b)
     Py_ssize_t n_items = asdl_seq_LEN(expr);
 
     const char* quote_str = PyBytes_AsString(a->bytes);
+    int triple_quote = strlen(quote_str) >= 4;
```

Comment:
Was wondering for a minute about why this is `>=4` instead of `>=3` but then noticed that `quote_str` also includes the prefix (like `f`, or `fr`), maybe let's add a comment here?